src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/transform/action/start/TransportStartTransformAction.kt

```diff
@@ -0,0 +1,150 @@
+package com.amazon.opendistroforelasticsearch.indexmanagement.transform.action.start
+
+import com.amazon.opendistroforelasticsearch.indexmanagement.IndexManagementPlugin.Companion.INDEX_MANAGEMENT_INDEX
+import com.amazon.opendistroforelasticsearch.indexmanagement.elasticapi.parseWithType
+import com.amazon.opendistroforelasticsearch.indexmanagement.transform.action.get.GetTransformAction
+import com.amazon.opendistroforelasticsearch.indexmanagement.transform.action.get.GetTransformRequest
+import com.amazon.opendistroforelasticsearch.indexmanagement.transform.action.get.GetTransformResponse
+import com.amazon.opendistroforelasticsearch.indexmanagement.transform.model.Transform
+import com.amazon.opendistroforelasticsearch.indexmanagement.transform.model.TransformMetadata
+import org.apache.logging.log4j.LogManager
+import org.elasticsearch.ElasticsearchStatusException
+import org.elasticsearch.ExceptionsHelper
+import org.elasticsearch.action.ActionListener
+import org.elasticsearch.action.DocWriteResponse
+import org.elasticsearch.action.get.GetRequest
+import org.elasticsearch.action.get.GetResponse
+import org.elasticsearch.action.support.ActionFilters
+import org.elasticsearch.action.support.HandledTransportAction
+import org.elasticsearch.action.support.master.AcknowledgedResponse
+import org.elasticsearch.action.update.UpdateRequest
+import org.elasticsearch.action.update.UpdateResponse
+import org.elasticsearch.client.Client
+import org.elasticsearch.common.inject.Inject
+import org.elasticsearch.common.xcontent.LoggingDeprecationHandler
+import org.elasticsearch.common.xcontent.NamedXContentRegistry
+import org.elasticsearch.common.xcontent.XContentHelper
+import org.elasticsearch.common.xcontent.XContentType
+import org.elasticsearch.rest.RestStatus
+import org.elasticsearch.tasks.Task
+import org.elasticsearch.transport.TransportService
+import java.time.Instant
+
+class TransportStartTransformAction @Inject constructor(
+    transportService: TransportService,
+    val client: Client,
+    actionFilters: ActionFilters
+) : HandledTransportAction<StartTransformRequest, AcknowledgedResponse>(
+    StartTransformAction.NAME, transportService, actionFilters, ::StartTransformRequest
+) {
+    private val log = LogManager.getLogger(javaClass)
+
+    override fun doExecute(task: Task, request: StartTransformRequest, actionListener: ActionListener<AcknowledgedResponse>) {
+        val getReq = GetTransformRequest(request.id(), null)
```

Comment:
minor: I see default values are set for srcContext in ``GetTransformRequest`` you don't need to explicitly pass ``null``. You can just do `GetTransformRequest(request.id())`